load_test.py

```diff
@@ -0,0 +1,211 @@
+import sys
+import os.path
+sys.path.append('../')
+from pathlib import Path
+import os, random, uuid, requests, argparse
+from app.main import ENDPOINTS 
+from env_config.env_config import EnvConfig
+from utils import to_oauth_request_form
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+BASE_PATH = Path(__file__).parent.resolve()
+
+# Fixed User Credentials
+user_credentials = {
+    "password": "Test1@123",
+    "password_confirm": "Test1@123",
+    "validation_link": "tabpfn"
+}
+
+def parse_args():
+    """
+    Parse command line arguments
+        1. Server URL
+        2. Number of total users
+            - default: 10 users
+        3. Number of parallel requests 
+            - default: 10 users hitting the server at the same time.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--server_url", 
+        type=str, 
+        default="https://tabpfn-server-wjedmz7r5a-ez.a.run.app",
+        help="Server URL"
+    )
+    parser.add_argument(
+        "--num_users", 
+        type=int, 
+        default=10,
+        help="Number of Users"
+    )
+    parser.add_argument(
+        "--num_requests", 
+        type=int, 
+        default=10,
+        help="Number of parallel requests"
+    )
+    return parser.parse_args()
+
+def generate_email():
+    """
+    Generate a random email address for a user: 
+    """
+    random_uuid = uuid.uuid4().hex.replace('%', '')
+    domain_name = random.choice(['gmail.com', 'outlook.org', 'yahoo.in', 'hotmail.com'])
+    random_email = "user" + f"{random_uuid}@{domain_name}"
+    return random_email
+
+def api_request(url, data=None, headers=None, type=0, files=None):
```

Comment:
Use an Enum for the Types, that will make the code much easier to understand.

Reply:
Sure! Thats cool.